README.md

```diff
@@ -1,46 +1,34 @@
-* [Documentation](https://docs.improbable.io/unreal/latest) (on the SpatialOS documentation website)
-* [Development roadmap](https://github.com/spatialos/UnrealGDK/projects/1) (Github project board)
-* Community: [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) - [Forums](https://forums.improbable.io/) -  [Mailing list](http://go.pardot.com/l/169082/2018-06-15/27ld2t)
-
-----
 ## Where to get the GDK and related projects
 The GDK and its related projects are available on GitHub.
 * [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
 * [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine)
 * [The Example Project](https://github.com/spatialos/UnrealGDKExampleProject) 
-* [Third Person Shooter project](https://github.com/spatialos/UnrealGDKThirdPersonShooter)
-* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)
 
 ## Unreal Engine changes
 In order to transform Unreal from a single server engine to a distributed model, we have made a number of small changes to the UE4 code. We will attempt to consolidate and remove (or submit as PR to Epic) as many of these changes as possible. You can see the changes in our forked [Unreal Engine repo](https://github.com/improbableio/UnrealEngine).
 
 > In order to get access to this fork, you need to link your GitHub account to a verified Epic Games account, and to have agreed to Epic's license. You will not be able to use the GDK for Unreal without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github).
 
 ## Recommended use
-The GDK is in [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) so we can react to feedback and iterate on development quickly. To facilitate this, during our alpha stage we don't have a formal deprecation cycle for APIs and workflows. This means that everything and anything can change. In addition, documentation is limited and some aspects of the GDK are not optimized.
+The GDK is in [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) so we can react to feedback and iterate on development quickly. To facilitate this, during our alpha stage we don't have a formal deprecation cycle for APIs and workflows. This means that everything and anything can change.
 
 We recommend using the GDK in projects in the early production or prototype stage. This ensures that your project's requirements are in line with the GDK's timeline.
 
-See the full [list of supported features](https://docs.improbable.io/unreal/latest/unreal-features-support) on the SpatialOS documentation website.
+## Versioning and support
+Please visit [this page](https://docs.improbable.io/unreal/alpha/content/pricing-and-support/versioning-scheme) for a description of the GDK's versioning scheme, and which branches to use when developing.
```

Comment:
this link will be live on wedesday when we release docs-next
it's currently correct in staging